### PR TITLE
#66: Added description of file format

### LINKED ISSUE #66: Specify file format

### PR DESCRIPTION
**Pull request by levBagryansky**
Closes #66

### CODE REVIEW

**Comment by levBagryansky** [comment]
@c71n93 please take a look, u are welcome to edit.

File: README.md [hunk 1]
```diff
@@ -11,15 +11,41 @@ $ build/chai <app.chai>
...
+  Ldia 8
+  Star r3
+  Ldra r3
+  Mul r2
+  Printc acc
+  Mov 0 r0
+  Ret
 ```
-Where Ret returns exit code `r0`.
+Where Ret returns exit code `r0`. "main" is name if the starting function, `8` is number of registers in it, `0` is number of arguments passing into the function.
```
Comment (c71n93) [inline README.md:25]: typo: "... is name **of** starting function"

**Review by c71n93 — APPROVED** [review]
Good! Only fix typo and you can merge!

**Comment by c71n93** [comment]
@levBagryansky oh, pdd is failed :(

**Pull request merged**